cloud/amazon/ec2_ami.py

```diff
@@ -87,6 +88,14 @@
     required: false
     default: null
     version_added: "2.0"
+  owners:
+    description:
+      - An optional list of owners of AMI image
+      - "It can be owner ID of image, special strings like 'self', 'amazon', 'aws-marketplace' or a combination of them"
```

Comment:
Valid values are your owner ID, 'self', 'amazon', 'aws-marketplace', or any combination of these options.